ext/bz2/bz2.stub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @return resource|false
+ */
+function bzopen (string $filename, string $mode) {}
+
+/**
+ * @param resource $bz
+ * @return string|false
+ */
+function bzread ($bz, $length = 1024) {}
+
+/**
+ * @param resource $bz
+ * @return int|false
+ */
+function bzwrite ($bz, $data, $length = null) {}
```

Comment:
```suggestion
function bzwrite ($bz, string $data, ?int $length = null) {}
```

Reply:
`$length` does not accept `null`; signature should be:
```suggestion
function bzwrite ($bz, string $data, int $length = UNKNOWN) {}
```

Reply:
it should include the optional identifier? like:

```suggestion
function bzwrite ($bz, string $data, ?int $length = UNKNOWN) {}
```

Reply:
Since it does not accept `NULL`, the nullable type declaration would be wrong (in other words, no question mark here). Thanks.